### ISSUE TITLE
Reach 100% test coverage

### ISSUE THREAD

**Issue by simonewebdesign** [simonewebdesign/elixir_pubsub#2]

**Comment by kinson** [comment]
Hello! I am practicing elixir for hacktoberfest and started working on this issue but cannot create a new branch/pr. Can you give me permission to add a branch?

**Comment by simonewebdesign** [comment]
Hi @kinson, please fork the project. At that point you'll be able to have your own branch, make your changes, and finally open a PR on this repo. :)

As a guidance, I'd really appreciate if you could test each function separately as doctest. Please read about it first if you're not familiar with it: https://elixir-lang.org/getting-started/mix-otp/docs-tests-and-with.html#doctests

The [current test] is not great. It'd be awesome if you could improve that part too.

So to recap, what I'd like here to happen is:
- to have every function unit tested independently in a doctest
- to have one (or more) end-to-end tests, in [test/pub_sub_test.exs][current test].

Let me know if you have any questions at any stage.

Thanks in advance!

[current test]: https://github.com/simonewebdesign/elixir_pubsub/blob/master/test/pub_sub_test.exs
